templates/webapps/galaxy/workflow/run.mako

```diff
@@ -583,9 +583,10 @@ if wf_parms:
                     <%
                     pja_ss_all = []
                     for pja_ss in [ActionBox.get_short_str(pja) for pja in step.post_job_actions]:
-                        pja_ss = h.escape( pja_ss )
                         for rematch in re.findall('\$\{.+?\}', pja_ss):
```

Comment:
Good points, it'd be easier to sanitize at get_short_str and trust that output.  I'll update the PR.